### PR TITLE
feat(19817): Introduce changes to prevent unexpected app closing

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Ticket title
Feature 19817
Implements changes which fix the root cause of the app closing unexpectedly.
https://jira.dvsacloud.uk/browse/CVSB-19817

## Checklist
- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by FBarrassDVSA — APPROVED** [review]

File: src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.spec.ts [hunk 1]
```diff
@@ -51,14 +51,18 @@ import { ActivityService } from '../../../../providers/activity/activity.service
 import { ActivityServiceMock } from '../../../../../test-config/services-mocks/activity-service.mock';
 import { LogsProvider } from '../../../../modules/logs/logs.service';
 import { AnalyticsService } from '../../../../providers/global';
+import { componentRefresh } from '@angular/core/src/render3/instructions';
```
Comment (Cullima) [inline src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.spec.ts:54]: can you remove these unused imports?

File: src/pages/testing/test-creation/test-review/test-review.ts [hunk 2]
```diff
@@ -88,9 +88,9 @@ export class TestReviewPage implements OnInit {
     private modalCtrl: ModalController,
     private alertCtrl: AlertController,
     private testResultService: TestResultService,
-    private openNativeSettings: OpenNativeSettings,
+    public openNativeSettings: OpenNativeSettings,
```
Comment (Cullima) [inline src/pages/testing/test-creation/test-review/test-review.ts:91]: why the need to make these public?

File: src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.ts [hunk 3]
```diff
@@ -61,7 +61,7 @@ export class VehicleLookupPage {
     private callNumber: CallNumber,
     public appService: AppService,
     private modalCtrl: ModalController,
-    private activityService: ActivityService,
+    public activityService: ActivityService,
```
Comment (Cullima) [inline src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.ts:64]: again, why are we making these public?

**Review by Cullima — CHANGES_REQUESTED** [review]

File: src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.spec.ts (hunk 1 above)
Comment (tomcrawleyy) [reply src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.spec.ts:54]: sure

File: src/pages/testing/test-creation/test-review/test-review.ts (hunk 2 above)
Comment (tomcrawleyy) [reply src/pages/testing/test-creation/test-review/test-review.ts:91]: it was unit test related, i'll look at reverting this as i managed to resolve it for the one below but had forgotten for this one

File: src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.ts (hunk 3 above)
Comment (tomcrawleyy) [reply src/pages/testing/vehicle/vehicle-lookup/vehicle-lookup.ts:64]: it was something id changed due to unit tests but figured out another way to do it, forgot to revert this

**Review by Cullima — APPROVED** [review]

**Pull request closed without merging**